src/DssSpell.sol

```diff
@@ -51,91 +45,112 @@ contract DssSpellAction is DssAction {
-    uint256 internal constant BILLION  = 10 ** 9;
-    uint256 internal constant WAD      = 10 ** 18;
-    uint256 internal constant RAD      = 10 ** 45;
+    uint256 constant internal MILLION  = 10 **  6;
 
-    // ---------- SBE parameter changes ----------
-    address internal immutable MCD_VOW            = DssExecLib.vow();
-    address internal immutable MCD_FLAP           = DssExecLib.flap();
+    address immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
 
-    // ---------- Reduce PSM-GUSD-A Debt Ceiling ----------
-    VatAbstract internal immutable vat            = VatAbstract(DssExecLib.vat());
+    // Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
 
-    // ---------- Increase RWA014-A (Coinbase Custody) Debt Ceiling ----------
-    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
 
     function actions() public override {
-        // ---------- Stability Fee Changes ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2
 
-        // Decrease the WBTC-A Stability Fee (SF) by 0.07%, from 5.86% to 5.79%
-        DssExecLib.setIlkStabilityFee("WBTC-A", FIVE_PT_SEVEN_NINE_PCT_RATE, /* doDrip = */ true);
 
-        // Decrease the WBTC-B Stability Fee (SF) by 0.07%, from 6.36% to 6.29%
-        DssExecLib.setIlkStabilityFee("WBTC-B", SIX_PT_TWO_NINE_PCT_RATE, /* doDrip = */ true);
+        // ---------- November Delegate Compensation ----------
+        // Forum: https://forum.makerdao.com/t/november-2023-aligned-delegate-compensation/23351
+
+        // 0xDefensor - 41.67 MKR - 0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9
+        // BONAPUBLICA - 41.67 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
+        // Cloaky - 41.67 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
+        // TRUE NAME - 41.67 MKR - 0x612F7924c367575a0Edf21333D96b15F1B345A5d
+        // BLUE - 13.95 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
+        // UPMaker - 13.89 MKR - 0xbB819DF169670DC71A16F58F55956FE642cc6BcD
+        // vigilant - 13.89 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
+        // JAG - 13.02 MKR - 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f
+        // PBG - 0.45 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
 
```

Comment:
nit, I suggest to put the note comment after each section instead of under all actions
(also without an empty line as it makes it unclear that the comment belongs to the comments above.)

(eg. [example](https://github.com/makerdao/spells-goerli/blob/dcff0085f3d8cfbb268980229a7b0812f5acdd9b/archive/2023-10-11-DssSpell/DssSpell.sol#L161-L164))